### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Shared Access Signature_ScreenReader:When the focus is on checkboxes present under permission label, if we perform any action(check/uncheck) on checkboxes same is not announced by Narrator 

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5714]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #Azure Tools Storage Explorer; #MAS1.3.1; #ScreenReader; #Narrator; #FTP; #Benchmark;

## Actual results:
When we are performing action like checking/unchecking the checkboxes, same is not announced by the narrator.

## Expected results:
Narrator should announce when we check/uncheck the checkbox.

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. right click on blob and select "Get Shared Access Signature" control and hit enter.
5. Turn on the Narrator in scan on mode.
6. Perform action on checkboxes present under "Permissions".
7. Verify the elements are MAS compliant.

## User Impact:
If screen reader doesn't announce the status of the checkboxes after performing the action, a screen reader might not know if the action is done.

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments

https://user-images.githubusercontent.com/105198219/168081453-6f6a231f-54b0-4188-ac85-f1348514a20d.mp4

**Comment by ManishaC1** [comment]
This bug is not reproduced on using NVDA. Hence closing it
